auction-server/src/state.rs

```diff
@@ -81,17 +85,10 @@ pub type BidAmount = U256;
 pub type GetOrCreate<T> = (T, bool);
 
 #[derive(Clone, Debug, ToSchema, Serialize, Deserialize)]
-#[schema(title = "BidResponse")]
-pub struct SimulatedBid {
+pub struct SimulatedBidShared {
```

Comment:
this name is kinda confusing. Maybe `AgnosticInfo`? `BidMetadata` would be my first choice, but that's already used elsewhere i think